uber/site_sections/groups.py

```diff
@@ -37,7 +37,8 @@ def form(self, session, new_dealer='', first_name='', last_name='', email='', me
             message = check(group)
             if not message:
                 session.add(group)
-                message = session.assign_badges(group, params['badges'], params['badge_type'])
+                ribbon_to_use = c.NO_RIBBON if 'ribbon' not in params else params['ribbon']
```

Comment:
Why do this like this? The checkbox should just pass NO_RIBBON, you shouldn't need to account for the ribbon not being in params.